content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md

```diff
@@ -114,3 +132,76 @@ See the [Gateway API](https://gateway-api.sigs.k8s.io/) documentation for inform
     HTTP/1.1 404 Not Found
     ...
     {{< /text >}}
+
+## Deployment methods
+
+In the example above, you did not need to install an ingress gateway `Deployment` prior to configuring a Gateway.
+In the default configuration, a gateway `Deployment` and `Service` is automatically provisioned based on the `Gateway` configuration.
+For advanced use cases, manual deployment is still allowed
+
+### Automated Deployment
+
+By default, each `Gateway` will automatically provision a `Service` and `Deployment` of the same name.
+These configurations will be updated automatically if the `Gateway` changes (for example, if a new port is added).
+
+These resources can be customized in a few ways:
+
+* Annotations and labels on the `Gateway` will be copied to the `Service` and `Deployment`.
+  This allows configuring things such as [Internal load balancers](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer) that read from these fields.
+* Istio offers a few additional annotations to configure the generated resources:
```

Comment:
Should this be `an additional annotation` since there is only 1?

Reply:
There will be more later but yes